web/requirements.txt

```diff
@@ -1,9 +1,9 @@
-Django==1.11.16
+Django==1.11.18
```

Comment:
Just to confirm we are updating Django here too. Is that a requirement or just convenience? Should we do these package changes separately?

Reply:
Doing it here was just a convenience. Github was warning me that the version of Django we were using had a vulnerability and that we should update it. 

You are correct that this should have been done separately, as is not related to this PR.